tests/stacks/ibm/multi-user/test_data/expected/apps_v1_deployment_kubeflow-pipelines-profile-controller.yaml

```diff
@@ -28,6 +28,11 @@ spec:
         - python
         - /hooks/sync.py
         env:
+        - name: KFP_VERSION
+          valueFrom:
+            configMapKeyRef:
+              key: appVersion
+              name: pipeline-upstream-install-config-c958fm776d
```

Comment:
Because of the kustomize issue: https://github.com/kubernetes-sigs/kustomize/issues/2609.
We need to stop using commonPrefix to make this configMapKeyRef correctly find pipeline-upstream-install-config and append the suffix.